### PR TITLE
Update docs for instance segmentation models

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
@snosov1 @DmitriySidnev

**Comment by druzhkov-paul** [comment]
@kchechil

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by kchechil** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build

File: intel_models/index.md [hunk 1]
```diff
@@ -102,8 +102,9 @@ instance instance segmentation model outputs pixel-wise masks for all instances.
...
 |------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
-| [instance-segmentation-security-0033](./instance-segmentation-security-0033/description/instance-segmentation-security-0033.md)                | 354.274              | 143.444    |
-| [instance-segmentation-security-0049](./instance-segmentation-security-0049/description/instance-segmentation-security-0049.md)                | 56.433               | 44.920     |
+| [instance-segmentation-security-0050](./instance-segmentation-security-0050/description/instance-segmentation-security-0050.md)                | 46.602               | 30.448     |
+| [instance-segmentation-security-0083](./instance-segmentation-security-0083/description/instance-segmentation-security-0083.md)                | 365.626              | 143.444    |
+| [instance-segmentation-security-0010](./instance-segmentation-security-0010/description/instance-segmentation-security-0010.md)                | 899.568              | 174.568    |
```
Comment (IRDonch) [inline intel_models/index.md]: Shouldn't 0010 be first?

Comment (druzhkov-paul) [reply intel_models/index.md]: Models are arranged by FLOPs/Params. Name is just a unique codename, nothing more than that.

**Pull request merged**